tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_survey_source_concept_id_test.py

```diff
@@ -0,0 +1,123 @@
+"""
+Integration test for update_survey_source_concept_id.py
+
+"""
+
+# Python imports
+import os
+
+# Third party imports
+from dateutil.parser import parse
+import pytz
+
+# Project imports
+from common import JINJA_ENV, VOCABULARY_TABLES
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.update_survey_source_concept_id import UpdateSurveySourceConceptId, DOMAIN_TABLES
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest\
+
+
+class UpdateSurveySourceConceptIdTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        # intended to be run on the rdr dataset
+        dataset_id = os.environ.get('RDR_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
+        cls.vocabulary_id = os.environ.get('VOCABULARY_DATASET')
+
+        cls.rule_instance = UpdateSurveySourceConceptId(project_id, dataset_id,
+                                                        sandbox_id)
+
+        sb_table_names = cls.rule_instance.get_sandbox_tablenames()
+        for table_name in sb_table_names:
+            cls.fq_sandbox_table_names.append(
+                f'{cls.project_id}.{cls.sandbox_id}.{table_name}')
+
+        for table_name in DOMAIN_TABLES + VOCABULARY_TABLES:
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}.{table_name}')
+
+        # call super to set up the client, create datasets, and create
+        # empty test tables
+        # NOTE:  does not create empty sandbox tables.
+        super().setUpClass()
+
+        # Copy vocab tables over to the test dataset
+        for src_table in cls.client.list_tables(cls.vocabulary_id):
+            destination = f'{cls.project_id}.{cls.dataset_id}.{src_table.table_id}'
+            cls.client.copy_table(src_table, destination)
```

Comment:
This works, but also feel free to use the [copy_vocab_tables](https://github.com/all-of-us/curation/blob/develop/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L337) class method.